src/App.tsx

```diff
@@ -1,8 +1,16 @@
 import { Provider } from "react-redux";
 import { PropsWithChildren } from "react";
 
+import { ErrorBoundary } from "react-error-boundary";
+
+import { Fallback } from "./features/components/fallback/fallback";
```

Comment:
rename to ErrorFallback and move to src/components, use import alias @/components/error-fallback

Reply:
resolved